### DISCUSSION TITLE
Bugs happening in subscribing Teams plan lead to double charge!!

### DISCUSSION THREAD

**Discussion by Jasmine-Yue** [community/community#34669] [Other Feature Feedback, Questions, & Ideas]
Hi,
Bug1: While subscribing to Team Plan, after I put the credit card info, it show 404 page. I thought I didn't  subscribe successfully. So I did it again, also changing a better name. It ended up I having two organization. 

 Bug2: Please make it obvious that we can choose monthly payment. Even we choose yearly plan at first without noticing the monthly plan, There is no place we can change the payment plan for this year. I have never saw something like this!!!

Bug3:  I am a student, I had to grant Ta to access to my repository. Since I accidently opened yearly plan, I had no way of choosing monthly pay for collaborator. I feel like I was robbed!!!!!!!!!!!!

I submitted two tickets in two days, no response! I guess when would I get the response!

**Comment by yiyuxin** [comment]
same to you
